### PR TITLE
W-13681362 1.5-fix-tls-docs-gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc [hunk 1]
```diff
@@ -1,47 +1,51 @@
...
+
+When you add a TLS context, you can also select the ciphers to use with the TLS context and configure different inbound and outbound validation settings.
 
 //end::intro1[]
+
 //tag::intro2[]
-You can format the YAML file to configure TLS either for a specific API Instance
-or or for all API Instances running on your Flex Gateway.
+Flex Gateway supports the following:
+
```
Comment (JennyHajee) [inline gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc:10]: Suggest adding periods for the bullet items.

File: gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc [hunk 2]
```diff
@@ -264,7 +268,7 @@ This example uses the default values for `alpn`, `minversion`, `maxversion`, and
 
 
 //tag::sample-config-outbound-instance-intro[]
-*Sample configuration for adding outbound TLS context for a specific API Instance:*
+*Sample configuration for adding outbound TLS context for a specific Upstream Service:*
```
Comment (JennyHajee) [inline gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc]: Should upstream service be lowercase?

File: gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc [hunk 3]
```diff
@@ -295,7 +299,7 @@ This example uses the default values for `alpn`, `minversion`, `maxversion`, and
 //end::sample-config-outbound-instance[]
 
 //tag::sample-config-mtls-outbound-intro[]
-*Sample configuration for adding outbound mTLS context for a specific API Instance::*
+*Sample configuration for adding outbound mTLS context for a specific Upstream Service:*
```
Comment (JennyHajee) [inline gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc]: Should upstream service be lowercase?

File: gateway/1.5/modules/ROOT/pages/flex-conn-tls-config.adoc [hunk 4]
```diff
@@ -58,7 +53,7 @@ include::anypoint-security::partial$task-flex-add-tls-context.adoc[tags=addConte
 
 You can apply a TLS context to an API instance either when creating a new instance or by editing an existing instance.
 
-API instance settings configuration has two parts, the downstream configuration and the upstream configuration. Inbound TLS context is a downstream configuration option, and Outbound TLS context is an upstream configuration option. When adding a new API instance, the downstream and upstream configuration is on separate pages. When editing an API instance, the downstream and upstream configuration options appear on the same page but are in different sections.
+API instance settings configuration has two parts, the downstream configuration and the upstream configuration. inbound TLS context is a downstream configuration option, and outbound TLS context is an upstream configuration option. When adding a new API instance, the downstream and upstream configuration is on separate pages. When editing an API instance, the downstream and upstream configuration options appear on the same page but are in different sections.
```
Comment (JennyHajee) [inline gateway/1.5/modules/ROOT/pages/flex-conn-tls-config.adoc]:
```suggestion
API instance settings configuration has two parts, the downstream configuration and the upstream configuration. Inbound TLS context is a downstream configuration option, and outbound TLS context is an upstream configuration option. When adding a new API instance, the downstream and upstream configuration is on separate pages. When editing an API instance, the downstream and upstream configuration options appear on the same page but are in different sections.
```

**Review by JennyHajee — APPROVED** [review]

File: gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc (hunk 1 above)
Comment (glenn-rodgers-sf) [reply gateway/1.5/modules/ROOT/pages/_partials/task-tls-config.adoc:10]: I'm not sure on this. The items themselves are fragments, but they complete the sentence. I don't think they should periods though.

From the style guide:

Follow these style guidelines when creating your bullet list:

- Limit list hierarchy to two levels: three at most. Check with your editor if you need help doing this.
- Capitalize the first letter of each item. Rework to not begin list items with terms that are case-sensitive.
- Make list items parallel. For example, make all list items a noun or all list items a phrase that starts with a verb.
- End list items with periods, unless the list is composed entirely of fragments,

**Pull request merged**